Cargo.toml

```diff
@@ -51,6 +52,7 @@ default = ["serde", "std"]
 legacy-ops = []
 maths = []
 maths-nopanic = ["maths"]
+rocket-fromform = ["rocket"]
```

Comment:
Maybe just `rocket-form`? It'd be nice to be able to extend it for other form features if they ever became available.

Reply:
What about `rocket-traits`? That should let us implement other stuff, such as [`FromParam`](https://api.rocket.rs/v0.5-rc/rocket/request/trait.FromParam.html).

Reply:
Even better. I like that.